src/index.tsx

```diff
@@ -587,6 +636,22 @@ const getAuthorizationToken = (context: SDSContext) => {
   ).then((data) => data.text());
 };
 
+const wrapSSML = (text: string, context: SDSContext) => {
+  if (["", " "].includes(text)) {
+    return new context.ttsUtterance("");
+  }
+  let content = `<speak version="1.0" xmlns="http://www.w3.org/2001/10/synthesis" xmlns:mstts="http://www.w3.org/2001/mstts" xml:lang="en-US"><voice name="${context.voice.name}">`;
```

Comment:
What does lang="en-US" do here? Just acts as a default?

Reply:
Should it be context.voice._name? Browser complains about it being missing

Reply:
No, this is should be correct -- it always worked this way. https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisVoice

I sometimes see this issue, but only with "customised" index.html where the context is overridden.